card.ml

```diff
@@ -22,6 +22,7 @@ type draw_penalty = int
 
 type t = {
   color : color;
+  digit : int option;
```

Comment:
Not sure if int is the best type - are you going to do any math with a digit? If not, maybe a variant would be more suitable